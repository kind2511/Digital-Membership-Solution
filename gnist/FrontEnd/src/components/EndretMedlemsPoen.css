.endret-medlems-poen-container {
    background-color: #f2f2f2;
    border: 1px solid #ccc;
    border-radius: 8px;
    padding: 20px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
  }
  
  .endret-medlems-poen-container .section-title {
    color: #333;
    font-size: 20px;
    margin-bottom: 10px;
  }
  
  