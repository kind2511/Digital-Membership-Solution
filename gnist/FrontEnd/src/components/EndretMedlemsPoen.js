import React from 'react';
import './EndretMedlemsPoen.css';

function EndretMedlemsPoen() {
    return (
        <div className="endret-medlems-poen-container">
            <h2 className="section-title">Endret medlems poeng</h2>
            <div className="section-content"></div>
        </div>
    );
}

export default EndretMedlemsPoen;
