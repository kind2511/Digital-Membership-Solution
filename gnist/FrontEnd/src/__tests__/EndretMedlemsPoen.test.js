import React from 'react';
import renderer from 'react-test-renderer';
import EndretMedlemsPoen from '../components/EndretMedlemsPoen';
import axios from 'axios';
import MockAdapter from 'axios-mock-adapter';

// Mock axios for the test
const mock = new MockAdapter(axios);
const membersMock = [
  {
    auth0ID: 'auth0|123456789',
    first_name: 'John',
    last_name: 'Doe',
    points: 100
  },
  {
    auth0ID: 'auth0|987654321',
    first_name: 'Jane',
    last_name: 'Doe',
    points: 200
  }
];
mock.onGet('http://127.0.0.1:8000/digital_medlemsordning/search_member/?name=').reply(200, membersMock);

test('EndretMedlemsPoen component snapshot', () => {
  const component = renderer.create(<EndretMedlemsPoen />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
