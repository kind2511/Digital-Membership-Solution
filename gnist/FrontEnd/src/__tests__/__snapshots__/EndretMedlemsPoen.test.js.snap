// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EndretMedlemsPoen component snapshot 1`] = `
<div
  className="endret-medlems-poen-container"
>
  <h2
    className="endret-medlems-poen-title"
  >
    Endret medlems poeng
  </h2>
  <input
    className="endret-medlems-poen-search-input"
    onChange={[Function]}
    placeholder="SÃ¸k etter medlem..."
    type="text"
    value=""
  />
  <div
    className="endret-medlems-poen-search-status"
  />
  <div
    className="endret-medlems-poen-section-content"
  />
</div>
`;
